settings.gradle

```diff
@@ -7,4 +7,14 @@ include 'DynamicListItems'
 include 'encryption'
 include 'FixedSizeTextColumns'
 include 'Fragments'
-include 'GsonJsonWebservice'
\ No newline at end of file
+include 'GsonJsonWebservice'
+include 'LeftNavBarLibrary'
+project(':LeftNavBarLibrary').projectDir = new File('LeftNavBarExample/LeftNavBarLibrary') 
```

Comment:
Basically both the `LeftNavBarLibrary` and `LeftNavBarExample` subprojects are related to the same example.
So I preferred to keep them in the same directory, but each one with its dedicated subdir